docs/reference/commandline/manifest.md

```diff
@@ -0,0 +1,277 @@
+---
+title: "manifest"
+description: "The manifest command description and usage"
+keywords: "docker, manifest"
+---
+
+<!-- This file is maintained within the docker/cli Github
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+## manifest
```

Comment:
This header isn't needed at all. We get this from the title in the page front-matter (line 2)